app/_hub/kong-inc/aws-lambda/1.0-x.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: AWS Lambda
 publisher: Kong Inc.
-version: 1.0.x
+version: 1.0-x
```

Comment:
why the "-" here?

Reply:
> why the "-" here?

@Tieske I was just matching the internal version with the file name and versions drop down menu. If these are wrong, that will have to be another PR.

![image](https://user-images.githubusercontent.com/3756245/96638828-5c955280-12e6-11eb-957e-9ad2c3a6ad8a.png)

Reply:
afaik only the Kong Enterprise packages were ever versioned with a -, eg. 0.34-1, etc. But that was dropped when the version got synced with CE. No plugin was ever versioned like that